### ISSUE TITLE
Weird IPs reported by tailscale netcheck

### ISSUE THREAD

**Issue by sylr** [tailscale/tailscale#5854]
### What is the issue?

I've got not IPv4 reported by tailscale netcheck and the IPv6 is reported to be [::ffff:1.2.3.4] where 1.2.3.4 is really my router's public IPv4.

Also I do really have a real IPv6 address on the device.

(1.2.3.4 is not really my public IP, it has been replaced for privacy).

### Steps to reproduce

```
$ tailscale netcheck
2022/10/06 15:25:17 portmap: [v1] Got PMP response; IP: 1.2.3.4, epoch: 1835802
2022/10/06 15:25:17 portmap: [v1] Got PCP response: epoch: 1835802
2022/10/06 15:25:17 portmap: [v1] UPnP reply {Location:http://10.0.0.1:46259/rootDesc.xml Server:Debian/wheezy UPnP/1.1 MiniUPnPd/2.0 USN:uuid:xxxx-aaaa-sssss-azeazeaze-qsdqsdd::urn:schemas-upnp-org:device:InternetGatewayDevice:1}, "HTTP/1.1 200 OK\r\nCACHE-CONTROL: max-age=120\r\nST: urn:schemas-upnp-org:device:InternetGatewayDevice:1\r\nUSN: uuid:xxxx-aaaa-sssss-azeazeaze-qsdqsdd::urn:schemas-upnp-org:device:InternetGatewayDevice:1\r\nEXT:\r\nSERVER: Debian/wheezy UPnP/1.1 MiniUPnPd/2.0\r\nLOCATION: http://10.0.0.1:46259/rootDesc.xml\r\nOPT: \"http://schemas.upnp.org/upnp/1/0/\"; ns=01\r\n01-NLS: 1663226915\r\nBOOTID.UPNP.ORG: 1663226915\r\nCONFIGID.UPNP.ORG: 1337\r\n\r\n"
2022/10/06 15:25:17 portmap: UPnP meta changed: {Location:http://10.0.0.1:46259/rootDesc.xml Server:Debian/wheezy UPnP/1.1 MiniUPnPd/2.0 USN:uuid:xxxx-aaaa-sssss-azeazeaze-qsdqsdd::urn:schemas-upnp-org:device:InternetGatewayDevice:1}

Report:
	* UDP: true
	* IPv4: (no addr found)
	* IPv6: yes, [::ffff:1.2.3.4]:60586
	* MappingVariesByDestIP:
	* HairPinning:
	* PortMapping: UPnP, NAT-PMP, PCP
	* Nearest DERP: Paris
	* DERP latency:
		- par: 11.7ms  (Paris)
		- lhr: 18.2ms  (London)
		- fra: 21.4ms  (Frankfurt)
		- ams: 24.9ms  (Amsterdam)
		- mad: 29.3ms  (Madrid)
		- waw: 39.1ms  (Warsaw)
		- nyc:         (New York City)
		- sfo:         (San Francisco)
		- sin:         (Singapore)
		- syd:         (Sydney)
		- blr:         (Bangalore)
		- tok:         (Tokyo)
		- dfw:         (Dallas)
		- sea:         (Seattle)
		- sao:         (São Paulo)
		- ord:         (Chicago)
		- den:         (Denver)
		- jnb:         (Johannesburg)
		- mia:         (Miami)
		- lax:         (Los Angeles)
		- hkg:         (Hong Kong)
		- tor:         (Toronto)
		- dbi:         (Dubai)
		- hnl:         (Honolulu)
```

### OS

macOS

### OS version

macOS 12.6

### Tailscale version

1.30.1 (App Store)

### Bug report

BUG-1aabf60c731a7870ca53ba0b452fc2c60dd64ba74a1e57b3c1869102c6e82873-20221006132926Z-4e8b1ea1b7c22333

**Comment by DentonGentry** [comment]
That is https://github.com/tailscale/tailscale/issues/5602, which is present in 1.30.1 but fixed in 1.30.2.
The App Store does a staged rollout over a 7 day period but says it is now offering 1.30.2 to 100% of users.
Does it offer to update Tailscale?

**Comment by sylr** [comment]
It does propose the update I missed it.

Upgrading does fix this. Thank you!
